src/logic/judge.ts

```diff
@@ -1,8 +1,15 @@
-export function isBoardFull(squares) {
+export enum PLAYER {
+    X = 'X',
+    O = 'O'
+}
+
+export type SQUARE = PLAYER | null;
```

Comment:
Wasn't sure of how to keep your abstraction here. So I created the type SQUARE, in the line below it would be weird to type
`isBoardFull(squares: PLAYER[])`